Dockerfile.Postgres

```diff
@@ -0,0 +1,3 @@
+FROM postgres:12.19-alpine
+COPY ./database/db.sql /docker-entrypoint-initdb.d/db.sql
```

Comment:
This docker image has been removed in a pull request wich has been merged today

Reply:
great, thanks!

When you refer to another PR in a comment, it is best to link to it explicitely.
This way, it will save my time when checking it. For that, just type '#' and then the number . In this case #23 I believe.

Reply:
noted, I'll do it next time